src/cf-layout.less

```diff
@@ -1605,6 +1605,121 @@
     }
 }
 
+/* topdoc
+  name: Featured content module
+  family: cf-layout
+  patterns:
+    - name: .featured-content-module
+      markup:
+        <section class="block
+                        block__border
+                        block__flush
+                        featured-content-module">
+          <div class="featured-content-module_text">
+            <p class=h4>
+              <span class="featured-content-module_icon cf-icon cf-icon-date">
+              </span>
+              Featured event
+            </p>
+            <h2>Consumer Advisory Board meeting</h2>
+            <p>Oct 22, 2015 @ 10:00AM EDT <br/>Washington, DC</p>
```

Comment:
This is just the markup example. Would it be clearer if we changed it to a non-event type of content. I'd be afraid people might think date/location is all that should go here.